### DISCUSSION TITLE
I claimed the namecheap free domain. but I dont want to use it on my repos

### DISCUSSION THREAD

**Discussion by NoobJsPerson** [community/community#121771] [GitHub Education]
### Hi everyone,

I have claimed the free namecheap domain with the Github Student Pack and it became the default domain all my github pages use. I tried removing the DNS records but that just broke it. Is there a way to get the old github pages domains or atleast add back the DNS records i removed?

**Comment by t4lhawi** [comment]
### Fixing Your GitHub Pages with Cloudflare

You've got your free Namecheap domain from the GitHub Student Pack, and now all your GitHub Pages are using it. But after removing some DNS records, things broke. No worries, we can fix this using Cloudflare! Here’s how:

#### Step 1: Sign Up for Cloudflare

1. **Create an Account**: Head over to [Cloudflare](https://www.cloudflare.com/) and sign up for a free account if you don’t have one.
2. **Add Your Domain**: Once you’re signed in, add your Namecheap domain to Cloudflare. They’ll guide you through the steps.

#### Step 2: Change Your Namecheap DNS Settings

1. **Log in to Namecheap**: Go to your Namecheap account.
2. **Go to Domain List**: Find the domain you got from the GitHub Student Pack and click “Manage”.
3. **Update Nameservers**:
   - Switch to the “Advanced DNS” tab.
   - Replace the existing nameservers with the ones Cloudflare gives you.

#### Step 3: Set Up DNS Records on Cloudflare

1. **Access DNS Settings**: In your Cloudflare dashboard, click on your domain and go to the “DNS” tab.
2. **Add Records**:
   - **A Record** for your root domain (like `example.com`):
     - Type: A
     - Name: @
     - IPv4 address: `185.199.108.153`
     - TTL: Auto
     - Proxy status: DNS only (turn off the proxy, it looks like an orange cloud).
   - **CNAME Record** for `www` (like `www.example.com`):
     - Type: CNAME
     - Name: www
     - Target: `your-github-username.github.io`
     - TTL: Auto
     - Proxy status: DNS only.

#### Step 4: Check DNS Propagation

- **Wait a Bit**: It might take a few minutes to a few hours for these changes to go through. You can use [DNSChecker](https://www.dnschecker.org/) to see if your DNS records are updated around the world.

#### Step 5: Adjust Your GitHub Pages Settings

1. **Remove Custom Domain**:
   - Go to your GitHub repository.
   - Click on “Settings” -> “Pages”.
   - Remove the custom domain if it's set.
2. **Re-add Custom Domain (if you want)**:
   - If you want your GitHub Pages to use your Namecheap domain, you can add it back in the same “Pages” settings.
3. **Check Your CNAME File**:
   - Make sure the `CNAME` file in your repository (if you have one) is correct. It should contain your custom domain name or be removed if you don’t want a custom domain.

**Reply by NoobJsPerson** [reply]
I already solved the problem. Thanks!

**Reply by shkshreyas** [reply]
how

**Reply by NoobJsPerson** [reply]
go to ur repo probably named shkshreyas.github.io and remove the domain from CNAME file

**Comment by t4lhawi** [comment]
hahahah, I just saw your comment now....
